packages/runtime-node/src/remote-node-entry.ts

```diff
@@ -0,0 +1,46 @@
+import { resolve, join } from 'path';
+import { parseCliArguments } from './parse-cli-arguments';
+import { ForkedProcess } from './forked-process';
+import { launchEngineHttpServer } from './launch-http-server';
+import { createIPC } from './process-communication';
+import type { ServerOptions } from 'socket.io';
+
+const isFirstArgumentPath = process.argv[1]!.startsWith('-');
```

Comment:
this is a bit naive. can't we have a proper argument parsing step somewhere?

Reply:
using minimist instead

Reply:
as discussed, make sure to use `parse([...])._`